types/swagger-schema-official/index.d.ts

```diff
@@ -55,18 +55,19 @@ export interface QueryParameter extends BaseParameter, BaseSchema {
   allowEmptyValue?: boolean;
 }
 
-export interface PathParameter extends BaseParameter {
+export interface PathParameter extends BaseParameter, BaseSchema {
   type: string;
   required: boolean;
 }
 
-export interface HeaderParameter extends BaseParameter {
+export interface HeaderParameter extends BaseParameter, BaseSchema {
```

Comment:
For proof:

<img width="356" alt="screen shot 2017-10-23 at 3 58 21 pm" src="https://user-images.githubusercontent.com/149178/31872866-156ff7ae-b80b-11e7-8ade-9966b1bdd859.png">